lib/sorbet-rails/model_plugins/active_record_enum.rb

```diff
@@ -13,20 +13,50 @@ def generate(root)
     model_class_rbi = root.create_class(self.model_class_name)
     model_class_rbi.create_include(enum_module_name)
 
+    enum_calls = ActiveRecordOverrides.instance.enum_calls[self.model_class_name]
+
     # TODO: add any method for signature verification?
     model_class.defined_enums.sort.each do |enum_name, enum_hash|
+      value_type = enum_hash.values.map(&:class).map(&:to_s).uniq
+
+      return_type = if value_type.length == 1
+        "T::Hash[T.any(String, Symbol), #{value_type.first}]"
```

Comment:
Looks like we'll run into some issue with ruby 2.3 Integer vs FixNum hmm. IDK what's the best way to handle that.

Reply:
hmmm. it looks like the tests work as expected, but fail because 2.3 sees them as a `Fixnum` and the tests have an `Integer`.

I'm also not completely sure how to fix this...

Reply:
We probably can manually change Fixnum -> Integer for Ruby 2.3